profiles/bicycle.lua

```diff
@@ -1,96 +1,12 @@
 api_version = 1
 
 -- Bicycle profile
-
 local find_access_tag = require("lib/access").find_access_tag
+local Set = require('lib/set')
+local Sequence = require('lib/sequence')
+local Handlers = require("lib/handlers")
+local next = next       -- bind to local for speed
```

Comment:
i read looking up global functions can be slow in lua. by assigning to a local it should be faster. the reason i bother in this case is that the function is used in the initial check for routabilitity, which is performed on every single way. haven't measured the effect though.